Makefile.vars

```diff
@@ -1,3 +1,4 @@
 QEMU_VERSION := 3.0.0
 LINUX_VERSION := 4.14.68
 BUSYBOX_VERSION := 1.29.3
+SWTPM_VERSION := 0.3.1
```

Comment:
This PR doesn't include Makefile changes to build or install swtpm in the stubdom image.

Reply:
@jandryuk yeah I thought I will go that way, but I neglected compilation of swtpm during initramfs creation, so this is useless change. What I do is:
1. compile swtpm to get deb packages
2. install in my system
3. copy binary and all dynamically linked libs to initramfs
This approach seem to work during development.